<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>472</width>
    <height>359</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>284</x>
      <y>200</y>
      <width>101</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>&amp;Push</string>
    </property>
    <property name="icon">
     <iconset resource="ch5.qrc">
      <normaloff>:/bottom.png</normaloff>:/bottom.png</iconset>
    </property>
   </widget>
   <widget class="QToolButton" name="toolButton">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>10</y>
      <width>91</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>AAA</string>
    </property>
    <property name="icon">
     <iconset resource="ch5.qrc">
      <normaloff>:/open.png</normaloff>:/open.png</iconset>
    </property>
    <property name="toolButtonStyle">
     <enum>Qt::ToolButtonTextUnderIcon</enum>
    </property>
   </widget>
   <widget class="QToolButton" name="toolButton_2">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>10</y>
      <width>81</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>BBB</string>
    </property>
    <property name="icon">
     <iconset resource="ch5.qrc">
      <normaloff>:/save.png</normaloff>:/save.png</iconset>
    </property>
    <property name="toolButtonStyle">
     <enum>Qt::ToolButtonTextUnderIcon</enum>
    </property>
   </widget>
   <widget class="QToolButton" name="toolButton_3">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>10</y>
      <width>81</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>CCC</string>
    </property>
    <property name="icon">
     <iconset resource="ch5.qrc">
      <normaloff>:/exit.png</normaloff>:/exit.png</iconset>
    </property>
    <property name="toolButtonStyle">
     <enum>Qt::ToolButtonTextUnderIcon</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>240</y>
      <width>101</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>&amp;Default</string>
    </property>
    <property name="icon">
     <iconset resource="ch5.qrc">
      <normaloff>:/edit_remove.png</normaloff>:/edit_remove.png</iconset>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>110</y>
      <width>100</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>select1</string>
    </property>
    <property name="icon">
     <iconset resource="ch5.qrc">
      <normaloff>:/player_play.png</normaloff>:/player_play.png</iconset>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">buttonGroup</string>
    </attribute>
   </widget>
   <widget class="QRadioButton" name="radioButton_2">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>140</y>
      <width>100</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>select2</string>
    </property>
    <property name="icon">
     <iconset resource="ch5.qrc">
      <normaloff>:/player_play.png</normaloff>:/player_play.png</iconset>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">buttonGroup</string>
    </attribute>
   </widget>
   <widget class="QRadioButton" name="radioButton_3">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>180</y>
      <width>100</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>select3</string>
    </property>
    <property name="icon">
     <iconset resource="ch5.qrc">
      <normaloff>:/player_play.png</normaloff>:/player_play.png</iconset>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">buttonGroup</string>
    </attribute>
   </widget>
   <widget class="QCheckBox" name="checkBox">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>110</y>
      <width>111</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>check1</string>
    </property>
    <property name="icon">
     <iconset resource="ch5.qrc">
      <normaloff>:/edit_add.png</normaloff>:/edit_add.png</iconset>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_2">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>140</y>
      <width>111</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>check2</string>
    </property>
    <property name="icon">
     <iconset resource="ch5.qrc">
      <normaloff>:/edit_add.png</normaloff>:/edit_add.png</iconset>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_3">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>170</y>
      <width>111</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>check3</string>
    </property>
    <property name="icon">
     <iconset resource="ch5.qrc">
      <normaloff>:/edit_add.png</normaloff>:/edit_add.png</iconset>
    </property>
   </widget>
   <widget class="QCommandLinkButton" name="commandLinkButton">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>100</y>
      <width>168</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>next</string>
    </property>
    <property name="icon">
     <iconset resource="ch5.qrc">
      <normaloff>:/1downarrow.png</normaloff>:/1downarrow.png</iconset>
    </property>
   </widget>
   <widget class="QCommandLinkButton" name="commandLinkButton_2">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>140</y>
      <width>168</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>previous</string>
    </property>
    <property name="icon">
     <iconset resource="ch5.qrc">
      <normaloff>:/1uparrow.png</normaloff>:/1uparrow.png</iconset>
    </property>
   </widget>
   <widget class="QDialogButtonBox" name="buttonBox">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>220</y>
      <width>176</width>
      <height>27</height>
     </rect>
    </property>
    <property name="standardButtons">
     <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>472</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="ch5.qrc"/>
 </resources>
 <connections/>
 <buttongroups>
  <buttongroup name="buttonGroup"/>
 </buttongroups>
</ui>
